catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash 
+
+# Install dependencies
```

Comment:
We should just check if the all the dependencies are available and fail if they aren't instead of trying to install them in the script. Probably with an error message telling the user which programs are needed

Reply:
Lemme put that in there real quick